src/features/defi/components/DefiModalHeader/DefiModalHeader.tsx

```diff
@@ -4,13 +4,18 @@ import { Text } from 'components/Text'
 
 type DefiModalHeaderProps = {
   headerImageSrc: string
+  headerImageWidth: string
   headerText: string | [string, Record<string, string>]
 }
 
-export const DefiModalHeader = ({ headerImageSrc, headerText }: DefiModalHeaderProps) => (
+export const DefiModalHeader = ({
+  headerImageSrc,
+  headerText,
+  headerImageWidth
+}: DefiModalHeaderProps) => (
   <>
     <Box textAlign='center'>
-      <Image src={headerImageSrc} width='68px' />
+      <Image src={headerImageSrc} width={headerImageWidth} />
```

Comment:
```suggestion
      <Image src={headerImageSrc} maxWidth={headerImageWidth} />
```

I think it exists right?